tests/cli/test_family.py

```diff
@@ -13,32 +13,49 @@
 
 
 @pytest.fixture
-def generate_cutoffs(tmp_path):
+def generate_cutoffs_dict(tmp_path):
```

Comment:
```suggestion
def generate_cutoffs_dict():
```

Reply:
You can also have this fixture reuse the `generate_cutoffs` fixture perhaps to generate the inner structure?